app/grandchallenge/workstation_configs/models.py

```diff
@@ -264,15 +264,45 @@ def save(self, *args, **kwargs):
 
 class WindowPreset(TitleSlugDescriptionModel):
     width = models.PositiveIntegerField(
-        validators=[MinValueValidator(limit_value=1)]
+        blank=True, null=True, validators=[MinValueValidator(limit_value=1)]
+    )
+    center = models.IntegerField(blank=True, null=True)
+
+    lower_percentile = models.PositiveSmallIntegerField(
+        blank=True, null=True, validators=[MaxValueValidator(limit_value=100)]
+    )
+
+    upper_percentile = models.PositiveSmallIntegerField(
```

Comment:
Could you add a check and test that `upper_percentile > lower_percentile`?

Reply:
Slightly outside the scope, but added form validation also.

Reply:
It's not "out of scope". When you're taking input from other sources it needs to be validated.